css/work-in-progress/microsoft/css-tables/visibility-collapse-col-002.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<meta charset="utf-8">
+<script src='/resources/testharness.js'></script>
+<script src='/resources/testharnessreport.js'></script>
+<link rel='stylesheet' href='base.css' />
+<link rel="author" title="Joy Yu" href="mailto:joysyu@mit.edu">
+<link rel="help" href="https://drafts.csswg.org/css-tables-3/#computing-the-table-width">
+<style>
+    x-table {
+      border: 5px solid black;
+    }
+    x-table span {
+        display: inline-block;
+        vertical-align: top;
+        background: lime;
+        margin: 3px;
+        height: 100px;
+        width: 100px;
+    }
+</style>
+<main>
+    <h1>Visibility collapse</h1>
+    <a href="https://drafts.csswg.org/css-tables-3/#computing-the-table-width">Spec</a>
+    <p>
+    Setting a column to visibility:collapse changes table width but not height.
+    </p>
+    <x-table id="one">
+      <x-tbody>
+        <x-tr>
+            <x-td>
+                <span>row 1</span>
+            </x-td>
+            <x-td>
+                <span></span>
+            </x-td>
+        </x-tr>
+        <x-tr>
+            <x-td>
+                <span>row 2</span>
+            </x-td>
+            <x-td>
+                <span></span>
+            </x-td>
+        </x-tr>
+        <x-tr>
+            <x-td>
+                <span>row 3</span>
+            </x-td>
+            <x-td>
+                <span style="height:50px"></span>
+            </x-td>
+        </x-tr>
+      </x-tbody>
+    </x-table>
+    Second table is identical to first except both column groups have been collapsed.
+    5px borders are still visible.
+    <x-table id="two">
+      <x-colgroup style="visibility:collapse;"></x-colgroup>
+      <x-colgroup style="visibility:collapse;"></x-colgroup>
+      <x-tbody>
+        <x-tr>
+            <x-td>
+                <span>row 1</span>
+            </x-td>
+            <x-td>
+                <span></span>
+            </x-td>
+        </x-tr>
+        <x-tr>
+            <x-td>
+                <span>row 2</span>
+            </x-td>
+            <x-td>
+                <span></span>
+            </x-td>
+        </x-tr>
+        <x-tr>
+            <x-td>
+                <span>row 3</span>
+            </x-td>
+            <x-td>
+                <span style="height:50px"></span>
+            </x-td>
+        </x-tr>
+      </x-tbody>
+    </x-table>
+
+</main>
+
+<script>
+    generate_tests(assert_equals, [
```

Comment:
In addition, I know that I initially spoke with @davidsgrogan and we planned to put them in WIP, and he had the good point of just putting them into CSS3 tables path. We should begin on doing this, the path is `css/css-tables-3`. I won't block on this as @FremyCompany or myself will move them a bit later. I also think it's imperative that we resolve on the spec language for the sizing and painting of `visibility: collapse` within [issue 478](https://github.com/w3c/csswg-drafts/issues/478). @FremyCompany and myself will try to land this, so if you could please stage these in the actual tables folder that would be greatly appreciated, but not required for my sign off. Thanks!